OpenSim/Tests/testXsensDataReader/testXsensDataReader.cpp

```diff
@@ -0,0 +1,93 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  exampleXsensDataReader.cpp            *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2019 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include "OpenSim/Common/DataAdapter.h"
+#include "OpenSim/Common/MapObject.h"
+#include "OpenSim/Common/XsensDataReader.h"
+#include "OpenSim/Common/STOFileAdapter.h"
+
+using namespace std;
+
+static void PrintUsage(const char *aProgName, ostream &aOStream);
+
+int main(int argc, char* argv[]) {
+    using namespace OpenSim;
+    if (argc < 4) {
+        PrintUsage(argv[0], cout);
+        return(-1);
+    }
+    std::string folder = "";
+    std::string trial = "MT_012005D6_025-000_"; //e.g. MT_012005D6_025-000_
+    std::string mappingFile = "mapIMUNamesToFilenames.xml";
+    for (int i = 1; i < argc; i++) {
+        string option = argv[i];
+        if (option == "-F")
+            folder = argv[i + 1];
+        else if (option == "-T")
+            trial = argv[i + 1];
+        else if (option == "-M")
+            mappingFile = argv[i + 1];
+    }
+    try {
+        MapObject mapXsensName2ModelName(mappingFile);
```

Comment:
This should fail to compile since `MapObject` no longer exists.

Reply:
That was a stale file unused otherwise, I removed it. Thanks for the find.